python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -76,8 +76,11 @@ def define_hermetic_runtime_toolchain_impl(
                 # tests for the standard libraries.
                 "lib/python{major}.{minor}*/**/test/**".format(**version_dict),
                 "lib/python{major}.{minor}*/**/tests/**".format(**version_dict),
-                # During pyc creation, temp files named *.pyc.NNN are created
+                # During pyc and pyo creation, temp files named *.pyc.NNN and *.pyo.NNN are created.
                 "**/__pycache__/*.pyc.*",
+                "**/__pycache__/*.pyo.*",
+                # File names with spaces should also be ignored.
```

Comment:
```suggestion
                # Ignore files with spaces because, while Bazel supports them,
                # the runfiles manifest format doesn't yet
```